server.js

```diff
@@ -38,6 +38,10 @@ hbs.registerHelper(lobars)
 hbs.registerAsyncHelper('octicon', async (data, cb) => {
   const { name, className, ariaLabel, width, height } = data.hash
 
+  if (data.hash.class) {
+    return console.log(`Octicons Helper: Use 'className' instead of 'class'.`)
```

Comment:
Perhaps use the same format as the error below, for consistency:

```suggestion
    return console.error(`ERROR(Octicons Helper): Use 'className' instead of 'class'.`)
```